backend/orchestrators/AdvancedOrchestrator.py

```diff
@@ -0,0 +1,271 @@
+import requests
+import os
+import copy
+import json
+import logging
+import openai
+from flask import Response, request, jsonify
+from .Orchestrator import Orchestrator
+from .LoadBalancer import LoadBalancer
+
+class AdvancedOrchestrator(Orchestrator):
+    # Post chat info if data configured
+    def conversation_with_data(self, request_body, message_uuid):
+        # Create a LoadBalancer object
+        load_balancer = LoadBalancer()
```

Comment:
Consider if this can/should be created during class init instead of for every chat request.

Reply:
Fixed.